src/AbstractNormalizedEventManager.php

```diff
@@ -13,6 +13,150 @@
 abstract class AbstractNormalizedEventManager extends AbstractWpEventManager
```

Comment:
The functionality that now facilitates WP compat is complicated, and IMHO deserves an extended explanation in class docs, or somewhere else.